rphys.sh

```diff
@@ -0,0 +1 @@
+#!/bin/sh
```

Comment:
Seems empty?

Reply:
The tutorial said to include it but I can delete if it's not needed.

Reply:
deleted it